tests/manifolds/test_symmetric_positive_definite.py

```diff
@@ -1,6 +1,6 @@
 import autograd.numpy as np
 import numpy.testing as np_testing
-from scipy.linalg import eigvalsh, expm
+from scipy.linalg import eigvalsh, expm, norm
```

Comment:
I know we do it in some places, but I've been trying to get rid of explicit imports lately and use the long form `scipy.linalg.eigvalsh`, etc. Also, why do you prefer scipy's `norm` here over numpy's?

Reply:
Ok for explicit imports. For scipy versus numpy, it was only a careless mistake, now corrected.